File: 1295-find-numbers-with-even-number-of-digits/1295-find-numbers-with-even-number-of-digits.cpp
class Solution {
public:
    int findNumbers(vector<int>& nums) {
        int ans=0;
        for(int i=0;i<nums.size();i++)
        {
            int temp=nums[i];
            int count=0;
            while(temp)
            {
                count++;
                temp/=10;
            }
            if(!(count&1)) ans++;
        }
        return ans;
        
    }
};

File: 1295-find-numbers-with-even-number-of-digits/1295-find-numbers-with-even-number-of-digits.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var findNumbers = function(nums) {
    return (nums.filter((ele)=>{
        // console.log(ele,toString(ele),toString(ele).length);
        return ((ele.toString().length) % 2 === 0);
    })).length;
};